lms/djangoapps/ccx/views.py

```diff
@@ -442,29 +467,33 @@ def ccx_student_management(request, course, ccx=None):
     action = request.POST.get('student-action', None)
     student_id = request.POST.get('student-id', '')
     user = email = None
+    error_message = ""
+    course_key = CCXLocator.from_course_locator(course.id, ccx.id)
     try:
         user = get_student_from_identifier(student_id)
     except User.DoesNotExist:
         email = student_id
+        error_message = validate_student_email(email)
+        if error_message is None:
+            error_message = _(
+                'Unregistered user. An email has been sent to id "{email}" to invite the user to register'
```

Comment:
Remove "id" from the sentence, it makes no sense. `An email has been sent to "{email}"` is clearer.